client/src/components/profile/components/Camper.js

```diff
@@ -115,6 +116,13 @@ function Camper({
             <FontAwesomeIcon icon={faAward} /> Top Contributor
           </p>
           <p className='text-center'>{joinArray(yearsTopContributor)}</p>
+          <p>
```

Comment:
```suggestion
          <p className='text-center'>
```

Adding the class would render it in the center.